README.md

```diff
@@ -54,6 +54,11 @@ and the prior precision is optimized with cross-validation `"gridsearch"`.
 After that, the resulting LA is used for prediction with
 the `"probit"` predictive for classification.
 
+> [!IMPORTANT]
+> In `optimize_prior_precision`, make sure to match the arguments with
+> the ones you want to pass in `la(x, ...)`. What's good during
+> prior precision tuning might not be good during the prediction itself.
```

Comment:
What do you mean by "What's good during prior precision tuning might not be good during the prediction itself."? Sounds confusing to me and the first sentence already makes the point; you could maybe add "for prediction" at the end of the first sentence.